torchao/quantization/prototype/mixed_precision/scripts/quant_model_size.py

```diff
@@ -0,0 +1,35 @@
+def quantized_model_size_in_bytes(num_elements, group_size, bit_zeropoint, bit_scale, x, y, A, B):
+    # Size for A-bit quantization layers
+    size_A_bit = x * (num_elements * A + num_elements // group_size * (bit_zeropoint + bit_scale))
+    
+    # Size for B-bit quantization layers
+    size_B_bit = y * (num_elements * B + num_elements // group_size * (bit_zeropoint + bit_scale))
+    
+    # Total quantized model size in bits
+    total_size_bits = size_A_bit + size_B_bit
+    
+    # Convert to bytes
+    total_size_bytes = total_size_bits / 8
+    
+    # Convert to gigabytes
+    total_size_gb = total_size_bytes / (1024 ** 3)
+    
+    return total_size_gb
+
+# Example usage
+num_elements = 250945664 #number of elements per Llama3 linear layer
+group_size = 32  # Example value, please adjust as needed
+bit_zeropoint = 2  # Example value, please adjust as needed
+bit_scale = 2  # Example value, please adjust as needed
```

Comment:
are these bytes?

Reply:
Yes these are in bytes. Have fixed this.